backend/src/test/java/de/bund/digitalservice/ris/norms/integration/adapter/output/database/DBServiceIntegrationTest.java

```diff
@@ -302,7 +306,9 @@ void itLoadsAllAnnouncementsFromDB() {
     final List<Announcement> announcements = dbService.loadAllAnnouncements();
 
     // Then
-    assertThat(announcements).containsExactly(announcement2, announcement1);
+    assertThat(announcements)
+      .usingRecursiveFieldByFieldElementComparatorIgnoringFields("importTimestamp")
```

Comment:
Same here as the comment above, though if you check it in the other test, maybe is not needed here